docs/rum_collection.md

```diff
@@ -0,0 +1,102 @@
+
+## Setup
+
+1. Add the Gradle dependency by declaring the library as a dependency in your `build.gradle` file:
+
+    ```conf
+    repositories {
+        maven { url "https://dl.bintray.com/datadog/datadog-maven" }
+    }
+
+    dependencies {
+        implementation "com.datadoghq:dd-sdk-android:x.x.x"
+    }
+    ```
+
+2. Initialize the library with your application context and your [Datadog client token][4]. For security reasons, you must use a client token: you cannot use [Datadog API keys][5] to configure the `dd-sdk-android` library as they would be exposed client-side in the Android application APK byte code. For more information about setting up a client token, see the [client token documentation][4]. You also need to provide an Application ID (see our [RUM Getting Started page][7]).
+
+    {{< tabs >}}
+    {{% tab "US" %}}
+
+```kotlin
+class SampleApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+        val config = DatadogConfig.Builder(BuildConfig.DD_CLIENT_TOKEN, BuildConfig.APPLICATION_ID)
+                        .trackGestures()
+                        .useViewTrackingStrategy(strategy)
+                        .build()
+        Datadog.initialize(this, config)
+    }
+}
+```
+
+    {{% /tab %}}
+    {{% tab "EU" %}}
+
+```kotlin
+class SampleApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+        val config = DatadogConfig.Builder(BuildConfig.DD_CLIENT_TOKEN, BuildConfig.APPLICATION_ID)
+                                             .trackGestures()
+                                             .useViewTrackingStrategy(strategy)
+                        .useEUEndpoints()
+                        .build()
+        Datadog.initialize(this, config)
+    }
+}
+```
+
+    {{% /tab %}}
+    {{< /tabs >}}
+    
+Depending on your application's architecture, you can choose one of several implementations of ViewTrackingStrategy :
+ 
+ - ActivityViewTrackingStrategy: every Activity in your application is considered a distinct View;
+ - FragmentViewTrackingStrategy: every Fragment in your application is considered a distinct View;
+ - NavigationViewTrackingStrategy: if you use the Android Jetpack Navigation library, this is the recommended strategy. It will automatically track Navigation destination as a distinct View;
+
+3. Configure and register the Rum Monitor. You only need to do it once, usually in your application's `onCreate()` method:
+
+    ```kotlin
+    val monitor = RumMonitor.Builder().build()
+    GlobalRum.registerIfAbsent(monitor)
+    ```
+   
+4. If you want to track your OkHttp requests as resources, you can add the provided [Interceptor][6] as follow:
+
+    ```kotlin
+    val okHttpClient =  OkHttpClient.Builder()
+        .addInterceptor(DatadogRumMonitor())
```

Comment:
```suggestion
        .addInterceptor(RumInterceptor())
```

Reply:
The DatadogInterceptor is recomended as it will automatically bundle RUM and Traces in the same interceptor.

Reply:
But you were having `DatadogRumMonitor` there not `DatadogInterceptor`

Reply:
Oups